src/components/hoc/withMeeting.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {concatMap} from 'rxjs/operators';
+import PropTypes from 'prop-types';
+import {AdapterContext} from '../hooks/contexts';
+
+/**
+ * withMeeting is a higher-order component that takes a component (WrappedComponent)
+ * and returns a new component (WithMeeting). The new component needs to be provided with a meeting destination
+ * on the "meetingDestination" prop. The wrapped component will receive the meeting object on the "meeting" prop.
+ *
+ * Example:
+ *
+ * class OriginalComponent extends Component {
+ *   render(props) {
+ *     return <div>Meeting id is {props.meeting.ID}</div>;
+ *   }
+ * }
+ *
+ * const EnhancedComponent = withMeeting(OriginalComponent);
+ *
+ * <EnhancedComponent meetingDestination="<MEETING_DESTINATION>" />
+ *
+ * @param {React.Component} WrappedComponent The component to wrap and enhance
+ * @returns {React.Component} The wrapped component
+ */
+
+/**
+ * @param WrappedComponent
```

Comment:
What's the purpose of this comment? shouldn't the above comment be the doc for the component?

Reply:
Hi @lalli-flores, here is my bad with this comment. I will remove that line and remain only the good comments. Thank you.